WindowManager/app/src/main/java/com/example/windowmanagersample/backend/ActivityExtensions.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.windowmanagersample.backend
+
+import android.app.Activity
+import android.graphics.Point
+import android.util.DisplayMetrics
+
+/**
+ * Return a [Point] whose dimensions match the metrics of the window.
+ * @return [Point] whose dimensions match the metrics of the window.
+ */
+@Suppress("DEPRECATION")
+internal fun Activity.calculateWindowSizeExt(): Point {
+    val displayMetrics = DisplayMetrics()
+
+    this.windowManager.defaultDisplay
+        .getMetrics(displayMetrics)
```

Comment:
Why aren't we using the currentWindowMetrics api here? Assuming it's the default display seems to be an area of possible issues. We can just use the currentWindowMetrics bounds in the Jetpack library instead of this extension method I believe?

Reply:
🤦 
I've removed altogether this extension and I'm using the WindowMetrics API in MidScreenFoldBackend